docs/tutorials/jupyter-sdk.rst

```diff
@@ -4,10 +4,16 @@ Using the Workflow SDK with Jupyter
 
 The Orquestra Workflow SDK can be used in Jupyter Notebooks!
 
+Prerequisites
+=============
+
+#. You've :doc:`installed Orquestra Workflow SDK<installing-macos-linux>`.
```

Comment:
Query: do we want to link the windows installation instructions here as well?

Reply:
So far we only linked to the macOS/Linux installation in our tutorials, because that's the main use case, and the macOS/Linux installation contains a link to the Windows one. I usually prefer conciseness over comprehensiveness.